.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  unit_tests:
+    name: Unit Tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9']
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x64
+
+    - name: Install pip dependencies
+      run: |
+       python -m pip install flake8 flake8-blind-except flake8-builtins flake8-class-newline flake8-comprehensions flake8-deprecated flake8-docstrings flake8-import-order flake8-quotes pytest
+       python -m pip freeze
```

Comment:
@clalancette maybe cache pip installed packages to speed things up a bit? See [actions/cache@v2](https://github.com/actions/cache/blob/main/examples.md#python---pip).

Reply:
Done in 7f8d743